src/main/kotlin/io/github/inductiveautomation/kindling/zip/views/ProjectView.kt

```diff
@@ -50,14 +50,16 @@ class ProjectView(override val provider: FileSystemProvider, override val path:
     override val icon: FlatSVGIcon = FlatSVGIcon("icons/bx-box.svg").derive(16, 16)
 
     companion object {
-        val exportZipFileChooser = JFileChooser(HomeLocation.currentValue.toFile()).apply {
-            isMultiSelectionEnabled = false
-            isAcceptAllFileFilterUsed = false
-            fileSelectionMode = JFileChooser.FILES_ONLY
-            fileFilter = FileNameExtensionFilter("ZIP Files", "zip")
+        val exportZipFileChooser by lazy {
```

Comment:
This has to be lazy to avoid a circular initialization issue - preferences is trying to load all the tools, but this tool is trying to look up a preference. It ends up throwing an NPE, despite Kotlin's guarantees, but just deferring creation of this chooser seems totally acceptable